ga4gh/datamodel/variants.py

```diff
@@ -209,6 +250,13 @@ def hashVariant(cls, gaVariant):
             str(tuple(gaVariant.alternateBases))).hexdigest()
 
 
+class AbstractVariantAnnotationSet(AbstractVariantSet):
```

Comment:
This class isn't being used at all --- we should probably just delete it, since an VariantAnnotationSet is a direct subclass of a VariantSet (or is it?).